repository/pom.xml

```diff
@@ -299,17 +299,6 @@
             <groupId>org.apache.poi</groupId>
             <artifactId>poi-ooxml</artifactId>
             <version>${dependency.poi.version}</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.apache.poi</groupId>
-                    <artifactId>poi-ooxml-schemas</artifactId>
```

Comment:
This transitive dependency is not even really there so I'm not sure why the exclude was here, probably by mistake.